platform/ios/export/export_plugin.cpp

```diff
@@ -1481,7 +1483,9 @@ Error EditorExportPlatformIOS::export_project(const Ref<EditorExportPreset> &p_p
 	String dest_dir = p_path.get_base_dir() + "/";
 	String binary_name = p_path.get_file().get_basename();
 
-	EditorProgress ep("export", "Exporting for iOS", 5, true);
+	bool export_project_only = p_preset->get("application/export_project_only");
+
+	EditorProgress ep("export", "Exporting for iOS", export_project_only ? 2 : 5, true);
```

Comment:
I'd suggest changing the export progress text to denote the "Export Project Only" option being enabled:

```suggestion
	if (export_project_only) {
		EditorProgress ep("export", "Exporting for iOS (Project Files Only)", 2, true);
	} else {
		EditorProgress ep("export", "Exporting for iOS", 5, true);
	}
	
```